src/main/proto/wfa/measurement/api/v2alpha/protocol_config.proto

```diff
@@ -47,11 +47,15 @@ message ProtocolConfig {
     LiquidLegionsSketchParams sketch_params = 1;
 
     // Differential privacy parameters for data provider noise registers.
-    // Required.  Immutable.
+    // Required. Immutable.
     //
     // Each of these registers contains a well-known constant register ID, and
     // arbitrary key and count values.
     DifferentialPrivacyParams data_provider_noise = 2;
+
+    // ID of the OpenSSL built-in elliptic curve. For example, 415 for the
+    // prime256v1 curve. Required. Immutable.
```

Comment:
Maybe add a comment, "currently only key size(what do you call this?) = 256 bit is supported." (our library is hardcoded to use 32+1 bytes as the size of a EC Point, It is easy to refactor the code, just low priority)